easybuild/easyconfigs/v/VTune/VTune-2021.5.0.eb

```diff
@@ -0,0 +1,27 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'VTune'
+version = '2021.5.0'
+
+homepage = 'https://software.intel.com/en-us/vtune'
+description = """Intel VTune Amplifier XE is the premier performance profiler for C, C++, C#, Fortran,
+ Assembly and Java."""
+
+toolchain = SYSTEM
+
+# By downloading, you accept the Intel End User License Agreement
+# (https://software.intel.com/content/www/us/en/develop/articles/end-user-license-agreement.html)
+# accept_eula = True
```

Comment:
@robert-mijakovic doesn't it need to be `--accept-eula-for=Intel-oneAPI`?

Reply:
```
rmijakovic@login01(13:28:10) (develop *%): ~/easybuild-easyconfigs/easybuild/easyconfigs $ grep -rn "accept-eula-for" *
rmijakovic@login01(13:28:35) (develop *%): ~/easybuild-easyconfigs/easybuild/easyconfigs $ 
```

Reply:
I think the option is called accept-eula-for and the variable is called accept_eula.